_rules/attr-not-duplicated-e6952f.md

```diff
@@ -55,31 +55,31 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
 
-No attributes are duplicated.
+This `img` element contains no duplicated attributes.
 
 ```html
 <img src="/test-assets/shared/w3c-logo.png" alt="W3C logo" />
 ```
 
 #### Passed Example 2
 
-No attributes, therefore no attributes are duplicated.
+This `br` element contains no attributes, so there is no duplicated attribute.
 
 ```html
 <br />
 ```
 
 #### Passed Example 3
 
-Empty attributes, no attributes are duplicated.
+This `input` element with a `type` attribute value of `checkbox` contains two different empty attributes.
```

Comment:
```suggestion
This `input` element contains three different attributes, two of them being empty.
```
* `type` is an attribute in itself.
* I don't think the actual attribute value is relevant in that rule.